src/components/DayPicker.jsx

```diff
@@ -104,14 +104,63 @@ export default class DayPicker extends React.Component {
   }
 
   getMonthHeightByIndex(i) {
-    const $transitionContainer = $(ReactDOM.findDOMNode(this.refs.transitionContainer));
-    return this.getMonthHeight($transitionContainer.find('.CalendarMonth').eq(i));
+    return this.getMonthHeight(
+      ReactDOM.findDOMNode(this.refs.transitionContainer).querySelectorAll('.CalendarMonth')[i]
```

Comment:
Not quite: https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#dom-node-refs